README.md

```diff
@@ -1,169 +1,79 @@
-to the 231 version of systemd (which is the one that Fedora and
-the other popular distributions are using in its stable releases).
-
-In order to do that, please use the following commands:
-
-```
-git clone git@github.com:coreos/systemd.git
-cd systemd
-git checkout v231
-./autogen.sh
-./configure
-make
-```
-
-You **shouldn't** do `make install` after that! Using the custom
-systemd-nspawn binary with the other components of systemd being
-in another version is totally fine.
-
-You may try to use master branch from upstream systemd repository, but
-it's very risky!
+## Requirements
 
-### Get needed Kubernetes repositories
+* **Host:** (tested with Fedora 26)
+  - systemd-nspawn v233
 
-kubeadm-nspawn needs the following repos to exist in your GOPATH:
+* **Container:** (tested with CoreOS Alpha)
+  - docker v17.05
 
-* [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes)
-* [kubernetes/release](https://github.com/kubernetes/release)
+* **Kubernetes:** 
+  - 1.7.0
 
-Also, bulding Kubernetes may rely on having your own fork and the
-separate remote called `upstream`. In this HOWTO, we assume that
-you have these repositories forked.
+## Quickstart
 
-You can clone then by the following commands:
+Make sure you have `glide` available in you PATH.
+In the directory where you cloned this repository, please do:
 
 ```
-mkdir -p $GOPATH/src/k8s.io
-cd $GOPATH/src/k8s.io
-git clone git@github.com:<your_username>/kubernetes.git
-git clone git@github.com:<your_username>/release.git
-cd kubernetes
-git remote add upstream git@github.com:kubernetes/kubernetes.git
-cd ../release
-git remote add upstream git@github.com:kubernetes/release.git
+# Get the needed CNI plugins
+$ go get -u github.com/containernetworking/plugins/plugins/main/bridge
+$ go get -u github.com/containernetworking/plugins/plugins/ipam/host-local
 ```
 
-### Build Kubernetes
-
-kubeadm-nspawn needs the built Kubernetes binaries and hyperkube
-Docker image. You need to build them like that:
+`kubeadm-nspawn` will configure the networks it needs in `/etc/cni/net.d`.
 
 ```
-cd $GOPATH/src/k8s.io/kubernetes
-build/run.sh make
-cd cluster/images/hyperkube
-make VERSION=latest
-```
+# Build the tool
+$ make vendor all
 
-### Get CNI plugins
+# Get the recommended container image
+$ sudo machinectl pull-raw --verify=no https://stable.release.core-os.net/amd64-usr/current/coreos_developer_container.bin.bz2 coreos
 
-```
-go get -u github.com/containernetworking/plugins/plugins/main/bridge
-go get -u github.com/containernetworking/plugins/plugins/ipam/host-local
-```
+# Spawn and provision nodes for the cluster
+$ sudo GOPATH=$GOPATH CNI_PATH=$GOPATH/bin ./kubeadm-nspawn --kubernetes-version=1.7.0 up --image=coreos --nodes=3
 
-`kubeadm-nspawn` will configure the networks it needs in `/etc/cni/net.d`.
+# Setup Kubernetes
+$ sudo GOPATH=$GOPATH CNI_PATH=$GOPATH/bin ./kubeadm-nspawn --kubernetes-version=1.7.0 init
+```
```

Comment:
I would be nice if the `Quickstart` section would include first kubectl commands and a verification it actually works (e.g. deploy a webserver pod, curl it).